routes/routes.js

```diff
@@ -44,7 +44,13 @@ const appRouter = function (app, gulp) {
       const prebidDownloadFilePath = path.resolve(prebidPath, 'build/dist', fileName);
       console.log(output);
       if(fs.existsSync(prebidDownloadFilePath)) {
-        res.download(prebidDownloadFilePath);
+        res.download(prebidDownloadFilePath, function (err) {
```

Comment:
I'm not an expert here so would like to kindly ask @mkendall07 to chime in.

The goal here is to remove generated file as soon as it is downloaded by the client and not needed anymore.